site/www/index.xslt

```diff
@@ -80,17 +80,19 @@ show(p)
                         <h4>Simplified Analysis</h4>
                         <p>Hail is an open-source Python library that simplifies genomic data analysis.
                         It provides powerful, easy-to-use data science tools that can be used to interrogate even
-                        biobank-scale genomic data (e.g UK Biobank, TopMed, FinnGen, and Biobank Japan).
+                        biobank-scale genomic data (e.g. UK Biobank, <a href="https://www.nature.com/immersive/d42859-020-00002-x/index.html">gnomAD</a>, TopMed, FinnGen, and Biobank Japan).
                     </p>
                     </section>
                     <section>
                         <h4>Genomic Dataframes</h4>
-                        <p>Modern data science is driven by table-like data structures, often called dataframes (see <a href="https://pandas.pydata.org">Pandas</a>).
-                            While convenient, they don't capture the structure of genetic data, which has row (variant) and
-                            column
-                            (genotype) groups.
-                            To remedy this, Hail introduces a distributed, dataframe-like structure called
-                            <a href="/docs/0.2/overview/matrix_table.html?highlight=matrix%20table" target="_blank">MatrixTable</a>.
+                        <p>Modern data science is driven by numeric matrices (see <a href="https://numpy.org/">Numpy</a>) and
+                           tables (see <a href="https://www.r-project.org/about.html">R</a> and <a href="https://pandas.pydata.org">Pandas</a>).
+                           While sufficient for many tasks, none of these tools adequately capture
+                           the structure of genetic data. Genetic data combines multiple axes (variants and samples) like
+                           matrices, and structured entries (genotypes) like tables or dataframes.
```

Comment:
I think this comma shouldn't be here.

Reply:
I struggled hard with this comma. I know it's not proper grammar, but also it looked confusingly verbose without. Will remove.